power_grid_model_c/power_grid_model/include/power_grid_model/auxiliary/dataset.hpp

```diff
@@ -104,6 +222,12 @@ template <dataset_type_tag dataset_type_> class Dataset {
     Buffer const& get_buffer(std::string_view component) const { return get_buffer(find_component(component, true)); }
     Buffer const& get_buffer(Idx i) const { return buffers_[i]; }
 
+    constexpr bool is_columnar(std::string_view component) const {
+        return is_columnar(find_component(component, true));
```

Comment:
I do not agree with the logic here. It throws an exception when the component is not found. This requires extra work later in the main model to check if the component exist or not.

My proposal: if the component is not found, always return false is `is_columnar`. The row-base `get_span` will automatically return an empty range so no extra handeling in main model is needed.

Reply:
Agree. Done.